### PR TITLE
New version: Telegram.TelegramDesktop version 3.0.0

### PR DESCRIPTION
**Pull request by vedantmgoyal9**
﻿- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [X] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/26545)

### CODE REVIEW

File: manifests/t/Telegram/TelegramDesktop/3.0.0/Telegram.TelegramDesktop.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# Created with YamlCreate.ps1 v2.0.0 using YAML parsing
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: Telegram.TelegramDesktop
+PackageVersion: 3.0.0
```
Comment (OfficialEsco) [inline manifests/t/Telegram/TelegramDesktop/3.0.0/Telegram.TelegramDesktop.installer.yaml:5]:
```suggestion
PackageVersion: 3.0
```

File: manifests/t/Telegram/TelegramDesktop/3.0.0/Telegram.TelegramDesktop.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,26 @@
+# Created with YamlCreate.ps1 v2.0.0 using YAML parsing
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: Telegram.TelegramDesktop
+PackageVersion: 3.0.0
```
Comment (OfficialEsco) [inline manifests/t/Telegram/TelegramDesktop/3.0.0/Telegram.TelegramDesktop.locale.en-US.yaml:5]:
```suggestion
PackageVersion: 3.0
```

File: manifests/t/Telegram/TelegramDesktop/3.0.0/Telegram.TelegramDesktop.yaml [hunk 3]
```diff
@@ -0,0 +1,8 @@
+# Created with YamlCreate.ps1 v2.0.0 using YAML parsing
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.0.0.schema.json
+
+PackageIdentifier: Telegram.TelegramDesktop
+PackageVersion: 3.0.0
```
Comment (OfficialEsco) [inline manifests/t/Telegram/TelegramDesktop/3.0.0/Telegram.TelegramDesktop.yaml:5]:
```suggestion
PackageVersion: 3.0
```

**Review by OfficialEsco — CHANGES_REQUESTED** [review]

File: manifests/t/Telegram/TelegramDesktop/3.0.0/Telegram.TelegramDesktop.yaml (hunk 3 above)
Comment (vedantmgoyal2009) [reply manifests/t/Telegram/TelegramDesktop/3.0.0/Telegram.TelegramDesktop.yaml:5]: Oops, sorry for the mistake.

**Review by OfficialEsco — APPROVED** [review]

**Pull request merged**